x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/DataFrameMessages.java

```diff
@@ -22,14 +22,16 @@
     public static final String REST_PUT_DATA_FRAME_FAILED_PERSIST_TRANSFORM_CONFIGURATION = "Failed to persist data frame configuration";
     public static final String REST_PUT_DATA_FRAME_FAILED_TO_DEDUCE_TARGET_MAPPINGS = "Failed to deduce target mappings";
     public static final String REST_PUT_DATA_FRAME_FAILED_TO_CREATE_TARGET_INDEX = "Failed to create target index";
+    public static final String REST_PUT_DATA_FRAME_TARGET_INDEX_ALREADY_EXISTS = "Target index [{0}] already exists";
```

Comment:
nit: would be good to be consistent and call it either destination or target, as the element is called `dest` I suggest destination.

Reply:
@hendrikmuhs, I agree, I said `target` here as it was what the other error messages had. I will change it (and the others) to use `dest`